src/containers/guest-home-page/reset-password-success/ResetPasswordSuccess.js

```diff
@@ -42,7 +42,7 @@ const ResetPasswordSuccess = ({ email }) => {
       />
 
       <TitleWithDescription
-        componentStyles={ style.wrapper }
+        componentStyles={ style }
```

Comment:
discard this change, we don't have to pass all styles, and get only needed

Reply:
fixed